miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppSecureStorage.kt

```diff
@@ -50,13 +50,12 @@ internal class MiniAppSecureStorage(private val activity: Activity) {
     fun secureStorageSize(
         miniAppId: String,
         onSuccess: (Long) -> Unit,
-        onFailed: (MiniAppSecureStorageError) -> Unit
     ) {
         if (isStorageAvailable(miniAppId)) {
             val sizeInBytes = File(secureStorageBasePath, "$miniAppId.txt").length()
             onSuccess(sizeInBytes)
         } else {
-            onFailed(MiniAppSecureStorageError.secureStorageUnavailableError)
+            onSuccess(0)
```

Comment:
So the default size is `0`?

Reply:
@rleojoseph yes for a non existance file, there is no file there so returning 0 is ok i think.